### PR TITLE
Add rubocop, hook up travis

### LINKED ISSUE #11: Add rubocop
Add [rubocop](https://github.com/rubocop-hq/rubocop) checking for tests.

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

Fixes #11;  hooks up travis so rubocop is run on PRs.

## Was the usage documentation (e.g. README) updated?

added travis build badge.

### CODE REVIEW

File: spec/support/download_helpers.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
+# frozen_string_literal: true
+
 module DownloadHelpers
   TIMEOUT = 10
   PATH    = Pathname.new(File.expand_path('../../downloads', __dir__))
 
-  extend self
+  module_function
```
Comment (jmartin-sul) [inline spec/support/download_helpers.rb]: after some quick reading up about `extend self` vs `module_function` (including a [rubocop issue about this rule](https://github.com/rubocop-hq/ruby-style-guide/issues/556)), i'm not sure i understand this change well enough to approve it.  but everything else LGTM.

**Comment by ndushay** [comment]
@mjgiarlo I did not run the tests;  i haven't done that before;  I may need some hand-holding (tomorrow)

**Comment by jermnelson** [comment]
The `reaccession_via_preassembly_spec` will continue to fail until pre-assembly on stage is back online and working.

**Comment by mjgiarlo** [comment]
@ndushay OK, wanna pair once pre-assembly is back up? (Working with ops to resolve that in the meantime.)

**Comment by ndushay** [comment]
@mjgiarlo currently failing exactly like master.  Merge and make a ticket to debug problem?

**Pull request merged**